templates/pages/entry.html

```diff
@@ -27,7 +27,19 @@
             <div class="row">
                 <!-- Review Image -->
                 <div class="col-md-6">
-                    <img src="http://placeimg.com/640/480/any" class="card-img" alt="alt image">
+                    <label class="image-selector" for="image-selector">
+                        <div class="overlay-container">
+                            <img src="http://placeimg.com/640/480/any" class="card-img" alt="alt image">
```

Comment:
I appreciate this is a placeholder image, but it will be important for accessibility that the `alt` attribute reflects the content of the image itself. A good resource on writing alternative texts: https://axesslab.com/alt-texts/.